### ISSUE TITLE
How can I use a single goal description for a whole task

### ISSUE THREAD

**Issue by yuchen-ji** [cliport/cliport#27]
Thanks for your solid work!
I have a little question about train model with a whole goal description:
I notice that you defined some single goal for whole task, e.g. 'pack all the blue and yellow boxes in the brown box'. Could i follow its setting to set another task, such as, replace the language template of the task `stack-block-pyramid-seq-seen-colors`, from 'put the {pick} block on {place}" to "stack all the blocks to form a pyramid'. Is this logically feasible?

**Comment by MohitShridhar** [comment]
@yuchen-ji, step-by-step goals are possible for some tasks like `stack-block-pyramid-seq-seen-colors`. But for some tasks like `packing-boxes-pairs-*` it's very hard to describe the intermediate steps. You have to come with something like "the third smallest cuboid looking thing should be placed on the top right corner below the block that's already there". At this point, the instruction is probably way too ambiguous and complicated. 

TLDR: for some tasks it's easy to generate step-by-step goals. For some tasks it's nearly impossible.

**Comment by yuchen-ji** [comment]
Thank you so much, could i use a single language description for task like `stack-block-pyramid-seq-seen-colors` instead of step-by-step goals.
